r53.tf

```diff
@@ -4,7 +4,7 @@ resource "aws_route53_record" "domain_amazonses_verification_record" {
   name    = "_amazonses.${var.domain}"
   type    = "TXT"
   ttl     = "3600"
-  records = ["${aws_ses_domain_identity.domain.verification_token}"]
+  records = ["${aws_ses_domain_identity.domain.verification_token}", "${var.ses_records}"]
```

Comment:
Actually, this probably won't work, have you tested it? You might need to use some interpolation function like `concat()`.

Reply:
yes, i've tested it... ;)

Reply:
![image](https://user-images.githubusercontent.com/3729585/38501286-e7eda7aa-3c0c-11e8-956a-b32d7bd1a7dd.png)

so you don't have to test it :)

Reply:
with ses_records = ["dsd", "dsda"]